README.md

```diff
@@ -1,3 +1,58 @@
 # Go-Bioportal
 
 Go client for the [Bioontology Bioportal](http://bioportal.bioontology.org/) API.
+
+## Tools
+
+### Bioportal to Neo4j
+
+Takes a standard Bioportal-based CSV file of a vocabulary and generates CSV files and a script to import using the `neo4j-import` tool. Child-parent relationships are related using `subClassOf` and class-vocabulary relationship is modeled as `classOf`.
+
+
+#### Install
+
+Requires Go to install, however if you want pre-built binaries [please request it](https://github.com/chop-dbhi/go-bioportal/issues/)!
+
+```
+go get github.com/chop-dbhi/go-bioportal/cmd/bioportal-to-neo4j
+```
+
+#### Usage
+
+The command takes a vocab ID, label, and the file. It will output a set of files and print a script to standard out. The paths in the script will likely need to be adjusted, but it should serve as a template.
+
+```
+bioportal-to-neo4j <vocab-id> <vocab-label> <vocab-file>
+```
+
+#### Example
+
+- Download the ICD10-CM vocabulary
+- Run the tool
+- Show the output
+
+```
+$ curl -L http://data.bioontology.org/ontologies/ICD10CM/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb&download_format=csv > icd10cm.csv
```

Comment:
Unsure if this is an issue for everyone but this curl hangs at "<http://purl.bioontology.org/ontology/STY/T203> rdfs:subClassOf <http://purl.bioontology.org/ontology/STY/T074> ."